internal/utils/validation.go

```diff
@@ -48,6 +49,19 @@ func ValidatePhoneNumber(phoneNumberStr string) error {
 	return nil
 }
 
+func ValidateStellarPublicKey(publicKey string) error {
+	if publicKey == "" {
+		return fmt.Errorf("public key cannot be empty")
+	}
+
+	_, err := keypair.ParseAddress(publicKey)
+	if err != nil {
+		return fmt.Errorf("the provided public key is not a valid Stellar public key")
+	}
+
+	return nil
+}
+
```

Comment:
I don't think you need this method. You can just rely on `strkey.IsValidEd25519PublicKey`, as used accross the project.

Reply:
Addressed in c174a217289def231177fad9bcfc56475f2200e7